src/framework/emqttb_worker.erl

```diff
@@ -280,6 +280,13 @@ model() ->
          , cli_short   => $A
          , target_node => [autorate]
          }}
+   , spawn_multiplier =>
```

Comment:
This parameter controls the number of workers, rather than behavior of the individual worker, so maybe it's not the best place for it.